### PR TITLE
add-kvm-wip

### PR DESCRIPTION
**Pull request by calvix**
let's have some testing release, shall we?

### CODE REVIEW

File: kvm.yaml [hunk 1]
```diff
@@ -1,3 +1,50 @@
...
+      version: 1.0.0
+    - name: cert-operator
+      version: 0.1.0
+    - name: cluster-operator
+      provider: kvm
+      version: 0.23.1
+    - name: flannel-operator
+      version: 0.2.0
+    - name: kvm-operator
+      version: 3.11.0-dev
```
Comment (calvix) [inline kvm.yaml]: just bumping this to dev

**Comment by tfussell** [comment]
I updated this from master, changed it to kvm-operator@3.10.1-dev, containerlinux@2345.3.1, and kubernetes@1.16.8. I also changed it to the next patch version (11.2.2) instead of minor (11.3.0) since we don't know what the change will be yet. Hope you don't mind.

**Comment by calvix** [comment]
I think a did not merge this because of someone was working on differnt release so i jsut abodonned.

**Comment by tfussell** [comment]
> I think a did not merge this because of someone was working on differnt release so i jsut abodonned.

Got it. Now that Ludacris is working on the next KVM release with versions from releases, I'm going to close this in favor of https://github.com/giantswarm/releases/pull/125 (probably the one you were referring to).

**Pull request closed without merging**